src/templates/bootstrap/html/form.ejs

```diff
@@ -1,4 +1,4 @@
-<{{ctx.tag}} class="{{ ctx.component.className }}" ref="html"
```

Comment:
Doesn't this mean that if they provide a "className" value in their component, that it will still be broken?  Shouldn't this be the following?

```
<{{ctx.tag}} class="formio-component-htmlelement {{ ctx.component.className }}"
```